server/boot/routes.js

```diff
@@ -111,4 +111,19 @@ module.exports = function(app) {
         });
       });
     });
+
+  //change password for the user
+  app.post('/request-password-change', function(req, res, next) {
+    User.find({where: {email: req.body.email}}, function(err, user) {
+    const userObj = user[0];
+    User.changePassword(userObj.id, req.body.oldPassword, 
+    req.body.newPassword, function(err, response) {
```

Comment:
IIUC this code correctly, anybody can change password to any user as log as they know user's email. That's a huge security vulnerability.

IMO, we should be invoking the built-in REST endpoint `POST /api/users/change-password` (see https://github.com/strongloop/loopback/pull/3299), which is already taking care of all security checks. The methods returns a JSON (or an empty) response, which is not exactly what we want in this web app. We can use afterRemote hook to send back an HTTP redirect pointing the browser to the page we want to display after the password was changed successfully.